### PR TITLE
Release 7.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+[Full Changelog](https://github.com/voxpupuli/beaker-rspec/compare/7.0.0...7.1.0)
+
+**Implemented enhancements:**
+
+- Reduce beaker-pe to a development dependendency [\#109](https://github.com/voxpupuli/beaker-rspec/pull/109) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Add acceptance tests to CI [\#110](https://github.com/voxpupuli/beaker-rspec/pull/110) ([ekohl](https://github.com/ekohl))
```
Comment (ekohl) [inline CHANGELOG.md:15]: For clarity, should this also mention beaker-pe was dropped altogether?

Comment (bastelfreak) [reply CHANGELOG.md:15]: good idea, I will update the PR title

Comment (bastelfreak) [reply CHANGELOG.md]: updated it. do you think this change quallifies for a major release? (I don't think so)

Comment (ekohl) [reply CHANGELOG.md]: I don't. It's easy enough for people to pull in if they need it.

**Review by ekohl — APPROVED** [review]

**Pull request merged**